MainModule/Server/Core/Remote.lua

```diff
@@ -855,48 +805,34 @@ return function(Vargs, GetEnv)
 				end
 			end;
 
-			Detected = function(p,args)
+			Detected = function(p: Player,args: {[number]: any})
 				Anti.Detected(p, args[1], args[2])
 			end;
 
-			TrelloOperation = function(p, args)
-				local adminLevel = Admin.GetLevel(p)
-				
-				local trello = HTTP.Trello.API
-				
-				local data = args[1]
-				if data.Action == "MakeCard" then
-					local command = Commands.MakeCard
-					if command and Admin.CheckComLevel(adminLevel, command.AdminLevel) then
-						local listName = data.List
+			TrelloOperation = function(p: Player,args: {[number]: any})
+				if Admin.GetLevel(p) > Settings.Ranks.Admins.Level then
+					local data = args[1]
+					if data.Action == "MakeCard" then
+						local list = data.List
 						local name = data.Name
 						local desc = data.Desc
-						
-						for _, overrideList in ipairs(HTTP.Trello.GetOverrideLists()) do 
-							if service.Trim(overrideList:lower()) == service.Trim(listName:lower()) then
-								Functions.Hint("You cannot create a card in that list", {p})
-								return
-							end
-						end
-						
+						local trello = HTTP.Trello.API(Settings.Trello_AppKey,Settings.Trello_Token)
 						local lists = trello.getLists(Settings.Trello_Primary)
-						local list = trello.getListObj(lists, listName)
+						local list = trello.getListObj(lists,list)
 						if list then
-							local card = trello.makeCard(list.id, name, desc)
-							Functions.Hint("Made card \""..card.name.."\" in list \""..list.name.."\"", {p})
+							local card = trello.makeCard(list.id,name,desc)
+							Functions.Hint("Made card \""..card.name.."\"",{p})
 							Logs.AddLog(Logs.Script,{
 								Text = tostring(p).." performed Trello operation";
 								Desc = "Player created a Trello card";
 								Player = p;
 							})
-						else
-							Functions.Hint("\""..listName.."\" does not exist", {p})
 						end
 					end
 				end
 			end;
```

Comment:
Why was TrelloOperation reverted? The changes that were made were security-related, it should not have been changed back